apps/ledger-live-mobile/src/components/CurrencyIcon.tsx

```diff
@@ -62,13 +67,26 @@ type Props = {
   radius?: number;
   bg?: string;
   circle?: boolean;
+  disabled?: boolean;
 };
 
-const CurrencyIcon = ({ size, currency, circle, color, radius, bg }: Props) => {
+const CurrencyIcon = ({
+  size,
+  currency,
+  circle,
+  color,
+  radius,
+  bg,
+  disabled,
+}: Props) => {
   const { colors } = useTheme();
+  const bgColor = useMemo(
+    () => ensureContrast(getCurrencyColor(currency), colors.constant.white),
```

Comment:
`getCurrencyColor` already uses `ensureConstrat` if you provide a second arg it looks like - maybe explain why `colors.constant.white` here ?

Reply:
I just copied the code from `ParentCurrencyIcon` which used the correct design, didn't look into the specifics